src/main/java/be/acara/events/service/pdf/PdfServiceImpl.java

```diff
@@ -63,6 +62,21 @@ public PdfServiceImpl(EventService eventService, QRCodeService qrCodeService) {
         return baos.toByteArray();
     }
 
+    /**
+     * generates the unique id for the ticket
+     * @param event the event for which the ticket is generated
+     * @param user the user that ordered the event
+     * @return a unique id
+     */
+    private String generateUniqueId(Event event, User user, int hash) {
+        String s = event.getName() + user.getUsername() + hash;
+        int num = 0;
+        for (int i = 0; i < s.length(); i++) {
+            num = 131 * num + s.charAt(i);
+        }
+        return Integer.toString(num);
+    }
```

Comment:
Generates an unique ID but when our app restarts, the unique ID changes.
Also, the generated ID was negative in both when I tested this.